scripts/install_elk.sh

```diff
@@ -0,0 +1,88 @@
+sudo add-apt-repository -y ppa:webupd8team/java
+sudo apt-get update
+sudo apt-get -y install oracle-java8-installer
+wget -O - http://packages.elasticsearch.org/GPG-KEY-elasticsearch | sudo apt-key add -
+echo 'deb http://packages.elasticsearch.org/elasticsearch/1.4/debian stable main' | sudo tee /etc/apt/sources.list.d/elasticsearch.list
+sudo apt-get update
+sudo apt-get -y install elasticsearch=1.4.4
+sed -i '/network.host/c\network.host\:\ localhost' /etc/elasticsearch/elasticsearch.yml
+sudo service elasticsearch restart
+sudo update-rc.d elasticsearch defaults 95 10
+cd ~; wget https://download.elasticsearch.org/kibana/kibana/kibana-4.0.1-linux-x64.tar.gz
```

Comment:
can you make a temp dir and download everything into it?  the rm -rf ~/kibana-4* worries me.